### ISSUE TITLE
Unraid docker container

### ISSUE THREAD

**Issue by steve1977** [Nezreka/Spotify-Plex-Playlist-Sync#1]
This reads really cool! Any chance to see an Unraid docker for this? Thanks for considering!

**Comment by Nezreka** [comment]
I think this would be a great idea. I'll look into this over the holiday.

**Comment by steve1977** [comment]
Great, thanks! Does this allows to sync both public and private playlists?

It's a one-way sync from spotify to plex, right?

**Comment by Nezreka** [comment]
Yep this will sync the playlist that you have on spotify to your plex server. If you update your spotify playlist you just need to run the app again to update your plex playlist.

**Comment by peterpiglet** [comment]
A docker for this tool would be great!
